modules/missingmotd.cpp

```diff
@@ -15,13 +15,14 @@
  */
 
 #include <znc/Modules.h>
+#include <znc/Client.h>
 
 class CMissingMotd : public CModule {
   public:
     MODCONSTRUCTOR(CMissingMotd) {}
 
     void OnClientLogin() override {
-        PutUser(":irc.znc.in 422 :MOTD File is missing");
+        PutUser(":irc.znc.in 422 " + GetClient()->GetNick() + " :MOTD File is missing");
```

Comment:
Yeah, but this will usually result in receiving a duplicate MOTD as the connected IRC server will also send an MOTD.

Is this module meant to block the servers MOTD? In that case, why not just use the block_motd module?